packaging/docker/custom/Dockerfile

```diff
@@ -2,8 +2,32 @@ ARG GRAFANA_VERSION="latest"
 
 FROM grafana/grafana:${GRAFANA_VERSION}
 
+USER root
+
+ARG GF_INSTALL_IMAGE_RENDERER_PLUGIN="false"
+
+RUN if [ $GF_INSTALL_IMAGE_RENDERER_PLUGIN = "true" ]; then \
+    echo "http://dl-cdn.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories && \
+    echo "http://dl-cdn.alpinelinux.org/alpine/edge/main" >> /etc/apk/repositories && \
+    echo "http://dl-cdn.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories && \
+    apk --no-cache update && \

```

Comment:
I don't think `--no-cache update` will make any difference, since there's no on-disk cache to update?

Reply:
Can try without them. Thought it was needed to read in the /etc/apk/repositories